app/views/channels.html

```diff
@@ -49,6 +49,7 @@
                   <td>
 
                     <div ng-if="channel.status!='deleted'">
+                      <button class="btn btn-info btn-xs" ng-click="duplicateChannel(channel)"><i class="glyphicon glyphicon-file"></i></button>
```

Comment:
I originally added in the tooltips for the three button to explain the role of each. Visually, this didn't look so great to me as there were too many tooltip popups for the buttons as i hovered over them. I can add them in to illustrate? or an alternative is to added the description in the column heading E.g "Duplicate/Edit/Delete"